gui/src/components/s3/account-management.vue

```diff
@@ -118,6 +118,15 @@
                     src="@/assets/actions/edit-green.svg"
                   />
                 </cortx-has-access>
+                <div
+                  v-if = "isLoggedInUserAdminOrManage()">
+                  <img
+                    id="iam-reset-password"
+                    v-on:click="onResetBtnClick(item)"
```

Comment:
There must be a tooltip on icon

Reply:
![image](https://user-images.githubusercontent.com/71690421/107800322-a8cc3d00-6d84-11eb-8bd5-4ded3f36943e.png)

Reply:
Tooltip added